Scripts/utils/validate_network.py

```diff
@@ -61,8 +61,26 @@ def validate(network, fares=None):
         intervals += param.official_node_numbers[modes]
     unofficial_nodes = set()
     for link in network.links():
+        if len(link.modes) == 0:
```

Comment:
This could be written as
```suggestion
        if not link.modes:
```

Reply:
This should be in some "extreme Python" lessons :D